modules/apps/portal-portlet-bridge/portal-portlet-bridge-soy-impl/src/main/resources/META-INF/resources/router/SoyPortletRouter.js

```diff
@@ -580,7 +579,7 @@ SoyPortletRouter.STATE = {
 	 * @type {string}
 	 */
 	element: {
-		setter: dom.toElement,
+		setter: (val) => document.querySelector(val),
```

Comment:
Don't care about this because we're removing it next month, but couldn't `val` be a `Node` here and this throw an exception?